src/pudl/transform/params/ferc1.py

```diff
@@ -3447,8 +3447,8 @@
                     "begin_yr_balance": "starting_balance",
                     "end_yr_balance": "ending_balance",
                     "report_prd": "report_prd",
-                    "end_qtr_bal": "end_qtr_bal",
-                    "pri_yr_q4_bal": "pri_yr_q4_bal",
+                    "end_qtr_bal": "ending_balance_qtr",
+                    "pri_yr_q4_bal": "starting_balance_qtr",
```

Comment:
The renamed `pri_yr_q4_bal` here is a bit misleading, since it does not pertain to the quarter that's specified in the record (Q4 of this year). Maybe we could just not rename the columns and not iterate over the column names, and use the original names in a multi-column assignment (can you do that?) in the loc `process_dbf()`?

Reply:
yeah this felt bad.  `ending_balance_qtr` feels better tbh but these are both temporary so i will revert.